packages/module/patternfly-docs/content/extensions/data-view/examples/Toolbar/Toolbar.md

```diff
@@ -89,7 +114,7 @@ The `useDataViewSelection` hook manages the selection state of the data view.
 
```

Comment:
again, I can't make GH native suggestions for lines 94-113: 

The data view toolbar can display a bulk selection component using `bulkSelect`. You can make use of a predefined [component group extension bulk select](/extensions/component-groups/bulk-select) component.

### Selection state
The `useDataViewSelection` hook manages the selection state of the data viwe. 

**Initial values:**
- Optional `initialSelected` array of record's identifiers selected by default 
- `matchOption` function to check if given record is selected
    - When no `matchOption` is passed, the `Array.prototype.includes()` operation is performed on the `selected` array.
**Return values:**
- `selected` array of currently selected records
- `isSelected` function returning the selection state for given record
- `onSelect` callback to modify the selection state and accepting `isSelecting` flag indicating if records are changing to selected or deselected and `items` containing affected records 

```js file="./SelectionExample.tsx"